FileUtil/FileUpload.py

```
@@ -0,0 +1,45 @@
+                status_code=status.HTTP_400_BAD_REQUEST,
+                detail="5MB가 넘는 파일은 저장할 수 없습니다."
+            )
+        #현재 파일이 전부 읽혀서 포인터가 맨 뒤로 가있으므로 다시 포인터를 맨 앞으로 옮긴다.
+        await file.seek(0)
+
+        if not os.path.exists(folder):
+            os.makedirs(folder)
+        filename = f"{uuid.uuid4()}_{file.filename}"
+        full_path = os.path.join(folder,filename)
+
+        with open(full_path,"wb") as f:
```

Ruff (0.14.11)

Async functions should not open files with blocking methods like `open`

(ASYNC230)